### PR TITLE
Feature/tailwind genesis theme

### PR DESCRIPTION
**Pull request by andrew-boyd**
This adds an exportable version of the Tailwind genesis theme. This will be used in the playground once `beta.13` launches to remove the inline theme and replace it with an imported variable.

### CODE REVIEW

**Comment by andrew-boyd** [comment]
They playground updates will occur in the `@formkit/vue2-live-code-editor` package independently but are themselves dependent upon the theme being an exportable item in `beta.13`

**Review by GustavoFenilli — COMMENTED** [review]
Looks good to me, people still can change what they want from it by changing the property.

File: packages/themes/src/tailwindcss/index.ts [hunk 1]
```diff
@@ -37,3 +38,7 @@ const FormKitVariants = plugin(function ({ matchVariant }) {
 })
 
 export default FormKitVariants
+export {
+  FormKitVariants,
+  genesis
+}
```
Comment (justin-schroeder) [inline packages/themes/src/tailwindcss/index.ts:44]:
```suggestion
}

```

File: packages/themes/src/tailwindcss/genesis.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    altLabel: 'block w-full mb-1 font-bold text-sm',
+    inner: '$reset inline-block mr-2',
+    input: 'peer absolute opacity-0 pointer-events-none',
+    innerLabel: 'text-[10px] font-bold absolute left-full top-1/2 -translate-x-full -translate-y-1/2 px-1',
+    thumb: 'relative left-0 aspect-square rounded-full transition-all w-5 bg-gray-100',
+    track: 'p-0.5 min-w-[3em] relative rounded-full transition-all bg-gray-400 peer-checked:bg-blue-500 peer-checked:[&>div:last-child]:left-full peer-checked:[&>div:last-child]:-translate-x-full peer-checked:[&>div:first-child:not(:last-child)]:left-0 peer-checked:[&>div:first-child:not(:last-child)]:translate-x-0',
+    valueLabel: 'font-bold text-sm',
+    wrapper: 'flex flex-wrap items-center mb-1'
+  }
+}
```
Comment (justin-schroeder) [inline packages/themes/src/tailwindcss/genesis.ts:140]:
```suggestion
}

```

**Review by justin-schroeder — CHANGES_REQUESTED** [review]

**Pull request merged**